apps/src/aichat/views/ChatMessageView.tsx

```diff
@@ -88,26 +88,30 @@ const ChatMessageView: React.FunctionComponent<ChatMessageViewProps> = ({
       ) : null;
   }
 
-  if (!isAssistant && !isChatHistoryView && assets && currentChannelId) {
-    footer = (
-      <div className={styles.assetRow}>
+  let header;
+  if (!isAssistant && assets && currentChannelId) {
+    header = (
+      <div className={styles.assetCol}>
         {assets.map(asset => {
           const filename = asset.filename;
-          return filename.endsWith('.pdf') ? (
-            <div key={filename} className={styles.pdfPreview}>
-              <FontAwesomeV6Icon
-                iconName="file-pdf"
-                className={styles.pdfIcon}
-              />
-              <span>{filename}</span>
-            </div>
-          ) : (
-            <img
-              key={filename}
-              alt=""
-              className={styles.imagePreview}
-              src={getAssetUrl(asset, currentChannelId, levelName)}
-            />
+          const url = getAssetUrl(asset, currentChannelId, levelName);
+          return (
+            <button
+              type="button"
+              className={styles.assetButton}
+              onClick={() => window.open(url, '_blank')}
+            >
+              {filename.endsWith('.pdf') ? (
+                <FilePreview type="pdf" filename={filename} url={url} />
```

Comment:
Do we want to click to open in new tab functionality on the staged assets as well? If so we could include the `<button>` wrapper inside the `FilePreview` component. Thoughts @dju90 @markabarrett?

Reply:
Bryan said no click to open on staged assets here: https://codedotorg.slack.com/archives/C06FELVTV0Q/p1742427166086209?thread_ts=1742412831.241269&cid=C06FELVTV0Q